torchvision/ops/_masks_to_bounding_boxes.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+
+def masks_to_bounding_boxes(masks: torch.Tensor) -> torch.Tensor:
+    """Compute the bounding boxes around the provided masks
+    The masks should be in format [N, H, W] where N is the number of masks, (H, W) are the spatial dimensions.
+    Returns a [N, 4] tensors, with the boxes in xyxy format
+    """
+    if masks.numel() == 0:
+        return torch.zeros((0, 4), device=masks.device)
+
+    h, w = masks.shape[-2:]
```

Comment:
Do you think it would be useful to also allow for tensors of shape (H, W), i.e. let the function accept a single mask instead of a batch?

Reply:
It would be useful to me if the tensors could have shape (H, W, D) or (batch, H, W, D) i.e. 3D images.

Reply:
I think it should be Tensor of [N, H, W] since all the other ops accept tensor of shape [N, 4].
Also (H, W) is a special case of [1, H, W] ? I don't think so we handle such cases where N = 1 differently for other ops.

Reply:
@NicolasHug IMO we should have a complimentary op for transforming labeled images (i.e. connected components like the image returned by functions like `scipy.ndimage.label` or `skimage.measure.label`) like `labeled_image_to_boxes`. What do you think?

Reply:
@RylanSchaeffer Good idea! I'll adapt it to 3D (or even n-D). But I suppose that can come in a future PR.